model/task_lifecycle.go

```diff
@@ -1009,7 +1009,7 @@ func RestartFailedTasks(opts RestartOptions) (RestartResults, error) {
 		if t.IsPartOfDisplay() {
 			displayTasksToCheck[t.DisplayTask.Id] = t.DisplayTask
 		} else if t.DisplayOnly {
-			displayTasksToCheck[t.Id] = t.DisplayTask
+			displayTasksToCheck[t.Id] = &t
```

Comment:
I'm not sure, but I worry this will end up with the same pointer for every value (because the t variable is the same for every iteration of the loop)

Reply:
Hm I guess pointers and loops are funny. I'll change it to not use a pointer.